capten/common-pkg/k8s/external_secret.go

```diff
@@ -126,14 +127,15 @@ func (k *K8SClient) CreateOrUpdateSecretStore(ctx context.Context, secretStoreNa
 func (k *K8SClient) CreateOrUpdateExternalSecret(ctx context.Context, externalSecretName, namespace,
 	secretStoreRefName, secretName, secretType string, vaultKeyPathdata map[string]string) (err error) {
 	secretKeysData := []ExternalSecretData{}
-	for key, path := range vaultKeyPathdata {
+	for path, key := range vaultKeyPathdata {
 		secretKeyData := ExternalSecretData{
```

Comment:
this fix not needed, issue with plugin config json file charts/kad/crossplane_plugin_config.json
fix secretKey and path which got exchanged in json